### PR TITLE
fix 'Unable to get property 'destUrl' of undefined or null reference' (close #1270)

### LINKED ISSUE #1270: [ScriptError]: Unable to get property 'destUrl' of undefined or null reference
We should return native `document.documentURI` value if this property does not supported.

Script for reproduce:
```js
var uri = document.documentURI;
```

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             documentURI: {
                 condition: domUtils.isDocument,
-                get:       doc => urlUtils.parseProxyUrl(doc.documentURI).destUrl,
+                get:       doc => doc.documentURI ? urlUtils.parseProxyUrl(doc.documentURI).destUrl : doc.documentURI,
```
Comment (LavrovArtem) [inline src/client/sandbox/code-instrumentation/properties/index.js]: I think you should move this condition to the `condition` part.
```javascript
condition: doc => doc.documentURI && domUtils.isDocument(doc),
```

Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: Agree

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Pull request merged**